library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/ComposedOptionSelectorDetector.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder
+
+import android.view.ViewGroup
+
+internal class ComposedOptionSelectorDetector(private val detectors: List<OptionSelectorDetector>) :
+    OptionSelectorDetector {
+    override fun isOptionSelector(view: ViewGroup): Boolean {
+        detectors.forEach {
+            if (it.isOptionSelector(view)) {
+                return true
+            }
+        }
+        return false
```

Comment:
can be even simpler I think:

```suggestion
        return detectors.any { it.isOptionSelector(view) }
```